recipes/ipytest/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "ipytest" %}
+{% set version = "0.5.0" %}
+{% set arch = "py2.py3" %}
+{% set fn = "{}-{}-{}-none-any.whl".format(name, version, arch) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/{{ arch }}/{{ name[0] }}/{{ name }}/{{ fn }}
+  sha256: c25e09654b2b431047ab7f381333188be2e8fc66146f1f2aed44dc105128b074
+  
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install {{ fn }} --no-deps --ignore-installed -vv "
+
+requirements:
+  build:
+    - python
```

Looking at the [build log](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=31170) (line 357 of "run docker build" on linux) it looks like this is Python 3 only?
```suggestion
    - python >=3
```

---

In PyPI, it's listed as python 2 or 3, but indeed "import ipytest" does fail for Python 2.7.  I'll go ahead and let the developer know that the package doesn't work with python 2.7 so he can either fix it or remove support.